gtsam/discrete/Potentials.cpp

```diff
@@ -27,8 +27,8 @@ using namespace std;
 namespace gtsam {
 
 // explicit instantiation
-template class DecisionTree<Key, double>;
-template class AlgebraicDecisionTree<Key>;
+//template class GTSAM_EXPORT DecisionTree<Key, double>;
+//template class GTSAM_EXPORT AlgebraicDecisionTree<Key>;
```

Comment:
Accidentally commented out these two?

Reply:
I make the template GTSAM_EXPORT so these two are not needed. Need to check if Windows is happy with this or not...

Reply:
Gotcha. If they are not needed then kill them.